metron-platform/metron-solr/src/main/scripts/create_collection.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+METRON_VERSION=${project.version}
+METRON_HOME=/usr/metron/$METRON_VERSION
+SOLR_VERSION=6.6.2
```

Comment:
I notice that we're depending on maven artifacts of 6.6.0 and we set the solr version to 6.6.2 here, is there a reason for that?  If we can align them, then we could replace this line with SOLR_VERSION=`${global_solr_version}`

This question applies for the rest of the shell scripts too

Reply:
Good catch and I agree they should match.  I think the mistake is actually the maven version.  We initially explored using HDP Search (which is version 6.6.2) but it's not quite ready yet so we went ahead with a manual approach in the meantime.  The thinking was this would make it easy to switch to HDP Search in the future.

Reply:
Latest commit should address this.